src/units/animation.cpp

```diff
@@ -1329,6 +1373,196 @@ void unit_animator::add_animation(unit_const_ptr animated_unit
 	animated_units_.AGGREGATE_EMPLACE(std::move(animated_unit), anim, text, text_color, src, with_bars);
 }
 
+bool unit_animator::move_unit_fake(int& index_movement_anim)
+{
+	const std::string tag = "move_unit_fake";
+	game_events::wml_action::map m = game_events::wml_action::registry();// m[tag] is call wml func by tag.
```

Comment:
Should be a const refence and not a copy, if you need the variable, or if not, just use `game_events::wml_action::registry()` directly.